### PR TITLE
add setup file and move all other files up one dir level

### PR DESCRIPTION
**Pull request by pgkirsch**
Closes issue #56 

Ready for review @whoburg @bqpd

### CODE REVIEW

File: gpkitmodels/aircraft/htailwb.py [hunk 1]
```diff
@@ -0,0 +1,100 @@
+from gpkit import Variable, Model
```
Comment (bqpd) [inline gpkitmodels/aircraft/htailwb.py]: err, I don't think you meant to commit this file?

File: gpkitmodels/aircraft/htailwb.py [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+    source: Hoburg, "Geometric Programming for Aircraft Design Optimization"
+    """
+
+<<<<<<< HEAD
+    def __init__(self, **kwargs):
+=======
+    def __init__(self, fwadd=0.4, Nlift=2.0, rh=0.75, rhocap=2700,
+              rhoweb=2700, sigmax=250E6, sigmaxshear=167E6, w=0.5):
+
+>>>>>>> wingboxrefactor
```
Comment (bqpd) [inline gpkitmodels/aircraft/htailwb.py]: it still has merge conflicts.

File: setup.py [hunk 3]
```diff
@@ -0,0 +1,51 @@
+"""Standard Python setup script for gpkit"""
```
Comment (bqpd) [inline setup.py]: ;)

File: gpkitmodels/aircraft/htailwb.py (hunk 1 above)
Comment (pgkirsch) [inline gpkitmodels/aircraft/htailwb.py]: Good catch you're right!

File: setup.py (hunk 3 above)
Comment (pgkirsch) [inline setup.py]: whoops

**Comment by bqpd** [comment]
Oops, now this has major merge conflicts after merging classmethods :(

**Comment by bqpd** [comment]
If it's just moving stuff, probably worth a new PR?

**Comment by whoburg** [comment]
huh weird -- this merges into master fine on my computer.

**Comment by bqpd** [comment]
Github's just confused, then?

**Comment by whoburg** [comment]
I'm going to try merging this on my computer and pushing to master. Wish me luck.

**Comment by whoburg** [comment]
merged manually.

**Pull request merged**